--- syscall_64.tbl	2021-11-25 13:45:20.393339659 +0000
+++ syscall2_64.tbl	2021-11-25 13:45:20.366672993 +0000
@@ -370,7 +370,7 @@
 446	common	landlock_restrict_self	sys_landlock_restrict_self
 447	common	memfd_secret		sys_memfd_secret
 448	common	process_mrelease	sys_process_mrelease
-
+449 common  kernel_2d_memcpy    sys_kernel_2d_memcpy
 #
 # Due to a historical design error, certain syscalls are numbered differently
 # in x32 as compared to native x86_64.  These syscalls have numbers 512-547.
